src/CattyTests/Firebase/FirebaseCrashlyticsReporterTests.swift

```diff
@@ -21,125 +21,98 @@
  */
 
 import Firebase
+
 @testable import Pocket_Code
+
 import XCTest
 
-final class FirebaseCrashlyticsSetupTests: XCTestCase {
+final class FirebaseCrashlyticsReporterTests: XCTestCase {
 
-    var app: AppDelegate?
     var crashlytics: CrashlyticsMock?
+    var reporter: FirebaseCrashlyticsReporter?
 
     override func setUp() {
         super.setUp()
 
         crashlytics = CrashlyticsMock(collectionEnabled: false)
-        app = AppDelegateMock(crashlytics: crashlytics!)
+        reporter = FirebaseCrashlyticsReporter(crashlytics: crashlytics!)
+        UserDefaults.standard.set(true, forKey: kFirebaseSendCrashReports)
     }
 
     func testSetupCrashReportsEnabled() {
-        UserDefaults.standard.set(true, forKey: kFirebaseSendCrashReports)
-
-        app?.setupCrashlytics()
+        _ = FirebaseCrashlyticsReporter(crashlytics: crashlytics!)
         XCTAssertTrue(crashlytics!.isCrashlyticsCollectionEnabled())
     }
 
     func testSetupCrashReportsDisabled() {
         UserDefaults.standard.set(false, forKey: kFirebaseSendCrashReports)
 
-        app?.setupCrashlytics()
+        _ = FirebaseCrashlyticsReporter(crashlytics: crashlytics!)
```

Comment:
When executed in a given, possible order this test will fail:

- assume a previous test called `crashlytics.setCrashlyticsCollectionEnabled(true)`
- `crashlytics` not initialized or reset when this is executed
- `crashlyticsCollectionEnabled` will still be enabled

My suggestion is to initialize a fresh instance of `crashlytics`.